Makefile

```diff
@@ -0,0 +1,89 @@
+###
+# This Makefile provides portable, stack-agnostic commands (unlike npm)
+#
+# These commands are meant to be run from your local machine, not from
+# within the docker container. Each command handles exec'ing into the
+# docker container to run the command
+
+ifneq (,)
+	This makefile requires GNU Make.
+endif
+# #######
+# Basics
+platform-onboarded:
+	make migrate-docker
+	docker exec -it platform_platform_1 /bin/bash
+	docker exec -it platform_platform_1 npm run dev
+
+platform-exec:
+	@docker exec -it platform_platform_1 bash
+exec:platform-exec
+
+# ######
+# DB
+
+
+seed-docker:
+	docker exec -it platform_platform_1 /bin/bash
+	composer migrate
+
+seed:
+	make seed-docker
+
+
+migrate:
+	make migrate-docker
+
+
+migrate-docker:
```

Comment:
this would have the effect of removing database contents? In that case I think we would need to give the task a scarier name, just like below ("db-total-reset")